parsing/location.ml

```diff
@@ -250,6 +251,110 @@ let get_pos_info pos =
   (pos.pos_fname, pos.pos_lnum, pos.pos_cnum - pos.pos_bol)
 ;;
 
+(* basic color handling *)
+module Color = struct
```

Comment:
This should go to `misc` (I think `misc/utils.ml` or `misc/terminfo.ml` would be reasonable; creating a new `misc/ansi_colors.ml` appears to be a nice choice, but there is a small risk of module conflicts among people linking compiler-libs), except for the `styles` stuff which makes sense in Location. `mark_{open,close}_tag` could either stay there or be parametrized over a `tag:string -> style list option` mapping.